Tutorial-ETK_thorn-Interpolation_to_Arbitrary_Grids_multi_order.ipynb

```diff
@@ -295,61 +302,66 @@
     "  // Set up output array:\n",
     "  // The name of the input gridfunction is always \"interp_arbgrid_MO_ETK::interped_gf\":\n",
     "  const CCTK_STRING input_array_names[1] = { \"interp_arbgrid_MO_ETK::interped_gf\" };\n",
-    "  CCTK_REAL *points_x,*points_y,*points_z,**output_f;\n",
+    "  CCTK_REAL *points_int_x,*points_int_y,*points_int_z; // Subset of points where to interpolate, restricted by out_of_bounds_xyz\n",
+    "  CCTK_REAL **output_int; // Output of interpolation, over restricted set of points\n",
+    "  CCTK_REAL **output_f; // Output to be written to dataset, filled with NaNs at points out of bounds\n",
+    "  CCTK_INT *interp_or_not; // Keeps track of which point are passed to the interpolator (1) or not (0)\n",
     "  // For benchmarking purposes:\n",
     "  time_t start_timer,end_timer;\n",
     "  time(&start_timer); // Resolution of one second...\n",
     "  CCTK_REAL time_in_seconds;\n",
     "\n",
-    "  int num_interp_points;\n",
+    "  int num_grid_points;\n",
+    "  int num_interp_points=0;\n",
     "  if(CCTK_MyProc(cctkGH)==0) {\n",
+    "      CCTK_REAL *points_x,*points_y,*points_z; // Full set of points of destination grid, including those out of bounds\n",
     "      // Step 1: Read list of desired interpolation destination points from file:\n",
     "\n",
     "      // Step 1.a: Read integer at top of file indicating number of points.\n",
-    "      int num_interp_pointsx,num_interp_pointsy,num_interp_pointsz;\n",
```

Comment:
It's not a good idea to call the destination grid "grid", because the grid we have access to in this thorn is the AMR (source) grid. Instead "grid" should be called "interp_dst_grid" something like that. Otherwise what grid it refers to is ambiguous.

Reply:
In other words there's more than one grid here. Let's remove ambiguity.